src/App.js

```diff
@@ -69,26 +69,32 @@ const App = () => {
             options={options}
         /> */}
 
-        <Routes>
-          <Route 
-            exact 
-            path="/" 
-            element={
-              <ProtectedRoute>
-                <HomePage />
-              </ProtectedRoute>
-            } 
-          />
-          <Route  
-            path="/login" 
-            element={<LoginPage />} 
-          />
-      </Routes> 
-      </InputContext.Provider>  
-      <Footer />
-    </>
+                <Routes>
+                    <Route
+                        exact
+                        path="/"
+                        element={
+                            <ProtectedRoute>
+                                <HomePage />
+                            </ProtectedRoute>
+                        }
+                    />
+                    <Route
+                        path="/login"
+                        element={
+                            //Uncomment protected route when authentication is done
 
-  );
+                            // <ProtectedRoute requiredAuthLevel='anonymous'>
+                            <LoginPage />
+                            // </ProtectedRoute>
+                        }
+                    />
+                    <Route path="/sign-up" element={<SignUp />} />
```

Comment:
According to the ticket this route should be inside `ProtectedRoute` with `requiredAuthLevel `set to `anonymous`. Could you please fix?
`Route should be a protected route (use ProtectedRoute component) with a requiredAuthLevel of 'anonymous'.`

Reply:
This still needs fixing, we went over it with Nikki and we will need it as is for now, the BE needs to do something about the protected route, but I am not sure what.